bucket-info/udot_testing_uris.txt

```diff
@@ -0,0 +1,85 @@
+URI
+gs://ut-udot-row-county-parcels/Beaver/:
```

Comment:
**issue**: remove the `gs://bucket-name/` so this isn't hard coded and we can have more flexibility